config/configmaps/install-log-regexes-configmap.yaml

```diff
@@ -199,6 +199,11 @@ data:
 
 
     # Generic OpenShift Install
+    - name: InconsistentResult
+      searchRegexStrings:
+      - "Error: Provider produced inconsistent result after apply"
+      installFailingReason: InconsistentResult
```

Comment:
InconsistentResult sounds a little generic. Can we call it InconsistentTerraformResult?

Reply:
done